main_test.go

```
@@ -0,0 +1,42 @@
+
+	redisUrl := fmt.Sprintf("redis://%s", s.Addr())
+	os.Setenv("ATMOS_REDIS_URL", redisUrl)
+
+	origDir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("failed to get current working directory: %v", err)
+	}
+	defer os.RemoveAll(path.Join(origDir, "testdata", "fixtures", "hooks-test", ".terraform"))
+	defer os.Chdir(origDir)
+
+	os.Chdir("testdata/fixtures/hooks-test")
```

golangci-lint (1.62.2)

Error return value of `os.Chdir` is not checked

(errcheck)